sentry-kotlin-multiplatform/src/androidMain/kotlin/io/sentry/kotlin/multiplatform/SentryBreadcrumb.kt

```diff
@@ -0,0 +1,114 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.Breadcrumb
+import io.sentry.kotlin.multiplatform.extensions.toAndroidSentryLevel
+import io.sentry.kotlin.multiplatform.extensions.toKMPSentryLevel
+
+actual class SentryBreadcrumb {
```

Comment:
Would be possible to be a `data` class? or since its a wrapper that would not make sense?